components/mdns/.cz.yaml

```diff
@@ -3,6 +3,6 @@ commitizen:
   bump_message: 'bump(mdns): $current_version -> $new_version'
   pre_bump_hooks: python ../../ci/changelog.py mdns
   tag_format: mdns-v$version
-  version: 1.4.3
+  version: 1.4.4
```

Comment:
Shouldn't this bump to 1.5.0?

Reply:
probably yes, as that subtype thing is considered a feature.

have changed to `v1.5.0`